modules/dynamic/src/smithy4s/dynamic/internals/DynamicModelCompiler.scala

```diff
@@ -256,9 +256,33 @@ private[dynamic] object Compiler {
             )
           }
 
-      val theEnum = stringEnumeration(values, values)
+      if (shape.traits.contains(IdRef("alloy#openEnum"))) {
+        // the runtime representation of normal enums is Int, but for open enums it's String to support arbitrary unknown values.
```

Comment:
**question:** should we go for a `String` representation for all the cases?

Reply:
I suppose that'd make sense.

Reply:
Although, int is fine